applications/CompressiblePotentialFlowApplication/tests/stochastic_body_fitted_opt/xmc_parameters.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
@riccardotosi How much can we simplify these parameters?

Reply:
Hmm I think I prefer to simplify as much as possible the json, I think it will help to understand what the case is doing

Reply:
Done in 2261fe3.